CMakeLists.txt

```diff
@@ -43,6 +43,9 @@ enable_language(CXX C)
 # Enable C++11 support universally, with C++14 where available
```

Comment:
Perhaps update the comment (or remove it)? This now looks like: "Use 14, and 17 where available"